test/common/index.js

```diff
@@ -68,12 +68,11 @@ if (process.argv.length === 2 &&
     !process.env.NODE_SKIP_FLAG_CHECK &&
     isMainThread &&
     hasCrypto &&
-    require.main &&
```

Comment:
Is there another way to know whether node was run with a file entry point? `require.main` doesn't exist in ESM and this change doesn't work for all tests.

Reply:
see https://github.com/tschaub/es-main

Reply:
Thanks, but I don't think I can use this.
`test/common/index.js` is not the entry point (and it's not ESM either). import chain is:

- `node test/parallel/something.mjs` (ESM)
- `import '../common/index.mjs'` (ESM)
- `import './index.js'` (CJS)

Can I find out from `common/index.js` that the entry point was a file (ESM or CJS, I don't need to know in this case)?

Example of an entry point that is not a file:
- `node -e "require('./test/parallel/sometest.js')"`

Reply:
I replaced `require.main` with `fs.existsSync(process.argv[1])`